sql-statements/sql-statement-create-database.md

```diff
@@ -10,25 +10,22 @@ aliases: ['/docs-cn/v2.1/sql-statements/sql-statement-create-database/','/docs-c
 
 ## 语法图
 
-**CreateDatabaseStmt:**
+```ebnf+diagram
+CreateDatabaseStmt ::=
+    'CREATE' 'DATABASE' IfNotExists DBName DatabaseOptionListOpt
 
-![CreateDatabaseStmt](/media/sqlgram/CreateDatabaseStmt.png)
+DatabaseSym ::=
+    DATABASE
```

Comment:
These lines are newly added. Please take a look~ @kennytm

Reply:
it should be `DatabaseSym ::= 'DATABASE' | 'SCHEMA'`, e.g. `CREATE SCHEMA ...` is a valid CreateDatabaseStmt.